### DISCUSSION TITLE
A lot of 404 on posts after migration from the old theme

### DISCUSSION THREAD

**Discussion by wojciehm** [CaiJimmy/hugo-theme-stack#1089] [Q&A] [answered]
Hey folks. 

After migration from another theme the URLs have been changed from https://www.domain.com/some-post.html to https://www.domain.com/p/some-post.html.

In my case I have following URL https://www.wojcieh.net/how-to-backup-vcenter-server-appliance-6-5/ which was working and it is gives 404 because it has been moved to https://www.wojcieh.net**/p/**how-to-backup-vcenter-server-appliance-6-5/.

How can I remove **/p/** from the URL?

<img width="973" alt="image" src="https://github.com/user-attachments/assets/bcafda58-6a94-4125-a793-4d5e383bf96d">

TiA.

BR, 
Wojciech

**Comment by ElastiCourse** [comment, marked as answer]
The URL structure is defined in hugo.yaml (or .toml). Have you removed the contents of the config when changing theme?
It should be a simple fix by adding these line to `hugo.yaml`:
```
permalinks:
    posts: /:slug/
    page: /:slug/
```

**Comment by wojciehm** [comment]
Thanks. This was it
